packages/x-components/src/x-modules/history-queries/components/clear-history-queries.vue

```diff
@@ -83,10 +83,30 @@
 
 The component exposes a single default slot, where you can add icons or text.
 
-```vue
-<ClearHistoryQueries>
-  <img class="x-history-query__icon" src="./my-icon.svg"/>
-</ClearHistoryQueries>
+```vue live
+<template>
```

Comment:
`<template>` can only have one root child. Wrap the components in a `div` for the code to work